disttask/framework/scheduler/scheduler.go

```diff
@@ -46,10 +47,11 @@ type InternalSchedulerImpl struct {
 }
 
 // NewInternalScheduler creates a new InternalScheduler.
-func NewInternalScheduler(ctx context.Context, id string, taskID int64, taskTable TaskTable, pool Pool) InternalScheduler {
-	logPrefix := fmt.Sprintf("id: %s, task_id: %d", id, taskID)
+func NewInternalScheduler(ctx context.Context, id string, ddlID string, taskID int64, taskTable TaskTable, pool Pool) InternalScheduler {
+	logPrefix := fmt.Sprintf("ddlid: %s, task_id: %d", ddlID, taskID)
 	schedulerImpl := &InternalSchedulerImpl{
 		id:        id,
+		ddlID:     ddlID,
```

Comment:
No usage of `ddlID`?

Reply:
In ut, we can use ddlId to identify which domain run the subtask.